src/search.cpp

```diff
@@ -347,7 +347,7 @@ void Search::Worker::iterative_deepening() {
                 // When failing high/low give some update (without cluttering
                 // the UI) before a re-search.
                 if (mainThread && multiPV == 1 && (bestValue <= alpha || bestValue >= beta)
-                    && elapsed() > 3000)
+                    && elapsed_time() > 3000)
```

Comment:
Added comments. I wouldn't say it's wrong, but if `elapsed` is used for PV output, info strings will flood at low depths (and this is why we have the time condition to prevent spam).